Duplicated condition in RepackMsg

```
/ddnet-source-steam/src/engine/client/client.cpp:134:9: warning: duplicated 'if' condition [-Wduplicated-cond]
    else if(MsgId == NETMSGTYPE_CL_SETTEAM)
         ^~
/ddnet-source-steam/src/engine/client/client.cpp:132:18: note: previously used here
    else if(MsgId == NETMSG_RCON_AUTH)
            ~~~~~~^~~~~~~~~~~~~~~~~~~
```
@ChillerDragon

--- a/src/engine/client/client.cpp
+++ b/src/engine/client/client.cpp
@@ -131,10 +131,6 @@ static inline bool RepackMsg(const CMsgPacker *pMsg, CPacker &Packer, bool Sixup
 				MsgId = protocol7::NETMSG_INPUT;
 			else if(MsgId == NETMSG_RCON_AUTH)
 				MsgId = protocol7::NETMSG_RCON_AUTH;
-			else if(MsgId == NETMSGTYPE_CL_SETTEAM)
-				MsgId = protocol7::NETMSGTYPE_CL_SETTEAM;
-			else if(MsgId == NETMSGTYPE_CL_VOTE)
-				MsgId = protocol7::NETMSGTYPE_CL_VOTE;
 			else if(MsgId == NETMSG_PING)
 				MsgId = protocol7::NETMSG_PING;
 			else

